File: PianoRiseApp/Models/Rol.cs
using System.ComponentModel.DataAnnotations;

namespace PianoRiseManagement.Models
{
    public class Rol
    {
        public int Id { get; set; }

        [Required]
        public string Nombre { get; set; } = null!;

        public ICollection<Usuario> Usuarios { get; set; } = new List<Usuario>();
    }
}

File: PianoRisePython/app/models/rol.py
from sqlalchemy import Column, Integer, String
from sqlalchemy.orm import relationship
from app import db

class Rol(db.Model):
    __tablename__ = "rol"

    id       = Column("id", Integer, primary_key=True)
    nombre   = Column("nombre", String(100), nullable=False)

    usuarios = relationship("Usuario", back_populates="rol", lazy="dynamic")
